cypress/e2e/invoice/invoice.spec.cy.js

```diff
@@ -0,0 +1,160 @@
+        cy.get('.mat-datepicker-toggle-default-icon').click()
+       // cy.get('.mat-calendar-body > .ng-star-inserted > .mat-calendar-body-active').click()
+       
+       cy.get('#mat-input-1').then(($date) => {
+          
+          const selectedDate = $date
+        
+          cy.get('.mat-calendar-body > .ng-star-inserted > .mat-calendar-body-active')
+            .click()
+            .then(() => {
+              const selectingDate = $date
+              
+              // make sure it's what we expected
+              expect(selectingDate).to.eq(selectedDate)
+          
+            })
+            
+        })
+        
+
+
+        // adding item name
+
+        cy.get('#mat-input-2').type('Item')
+        cy.get('#mat-input-2').should('have.value', 'Item').and('be.visible')
+
+        // adding quantity
+        cy.get('#mat-input-3').type('1').should('be.visible')
+        
+
+        // adding price
+        cy.get('#mat-input-4').type(5).should('be.visible')
+        
+        // subTotal
+        cy.get(':nth-child(6) > .col-md-2 > .text-right > span').invoke('text').as('subTotal').then(parseFloat).should('be.gte', 5)
+
+        // adding tax
+        cy.get('.mr-1').click()
+        cy.get('#mat-input-5').type('10').should('be.visible')
+
+        // adding shipping
+        cy.get('#AddNewItemShipp').click()
+        cy.get('#mat-input-6').type(5).should('be.visible')
+
+
+        // saving the total amount
+        cy.get(':nth-child(11) > .col-md-2 > .text-right > span').invoke('text').as('total')
+
+        cy.get('.btn-white').click()
+        cy.wait(5000)
+
+        // to Check the details on the popup
+        cy.get('.modal-title').contains('Successfully Added')
+
+        cy.get('.phoneNumber').then(($el) => {
+            $el.get('+923244323448')
+          })
+          cy.get('.headingInModal').then(($el) => {
+            $el.get('string')
+          })
+          
+
+
+    })
+
+    // test Case #2: Checking the pop with Correct Details
+
+    it('check the invoice total value on create and listing page',()=>{
+        
+
+        cy.get('table > tbody > :nth-child(1) > .cdk-column-totalValue').and('not.be.empty').then(($price) => {
+          // capture what num is right now
+          const lsitingTotalValue = parseFloat($price.text())
+        
+          cy.get(':nth-child(1) > .cdk-column-invoiceId > .circle-name-text')
+            .click()
+            .then(() => {
+              // now capture it again
+              const detailsTotalValue = parseFloat($price.text())
+        
+              // make sure it's what we expected
+              expect(detailsTotalValue).to.eq(lsitingTotalValue)
+            })
+            
+        })
+        
+    })
+
+    // test Case #3: Verify the Export button is working
+
+    it('verify the export button is working',()=>{
+     
+      cy.get('.outline-btn')
+      cy.get('.outline-btn')
+      .click()
+      .should('contain','Export')
```

Comment:
@AmnaShahid96 no need for this assertion, we need to click on that button whatever its contained text.